packages/constants/src/routes.ts

```diff
@@ -40,7 +40,8 @@ const routes = {
         dsaInterviewSheet: "/interview-prep/dsa-interview-questions"
     },
     // PrepYatra
-    prepYatra: "https://prepyatra.theboringeducation.com/",
+    
```

Comment:
Are you sure this key is not used anywhere else because it'll start directing to /prepyatra only